src/node_buffer.cc

```diff
@@ -310,6 +310,70 @@ void Base64Slice(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+// bytesCopied = buffer.mask(mask, target, targetOffset, sourceStart, sourceEnd)
+void Mask(const FunctionCallbackInfo<Value> &args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  uint32_t mask32 = args[0]->Uint32Value();
+  char* mask = (char*)&mask32;
```

Comment:
Use reinterpret_cast, don't use C-style casts.

I'm not sure if you're aware of this but this would have been pointer aliasing with any other pointer type.